init.fish

```diff
@@ -0,0 +1,14 @@
+function zoxide-add --on-event fish_prompt
+    if command -q zoxide
+        zoxide add
+    end
+end
+
+if command -q zoxide
+    abbr -a zi "z -i"
+    abbr -a za "zoxide add"
+    abbr -a zq "zoxide query"
+    abbr -a zr "zoxide remove"
+
+    bind \ez 'z -i'
```

Comment:
While this is definitely a useful keybinding, it would be inconsistent across shells. I think we should let users create their own customizations, rather than have a preinstalled keybinding.

Reply:
Ok, makes sense. Another way of doing this in fish world is to have a universal variable like `$ZOXIDE_KEY_BINDINGS` set to 0 by default. If the user sets it to 1 (`set -U ZOXIDE_KEY_BINDINGS 1) the key binding is turned on.

What do you think would be the best approach here?

Reply:
Either way, it still results in an inconsistency across different shells. I think custom keybindings should be left upto the user for now.